app/src/main/java/com/peacecorps/pcsa/MainActivity.java

```diff
@@ -60,6 +60,7 @@
     public static String FRAGMENT_TAG = MainActivityFragment.TAG;
     private FragmentManager fragmentManager = getSupportFragmentManager();
     SharedPreferences sharedPreferences;
+    private int last_expanded_group=-1;
 
```

Comment:
@codingblazer is this the correct naming convention for variables? Please make it camel case so that it is consistent with other variable names in the app.